src/lib/ComponentLoader.coffee

```diff
@@ -112,7 +112,7 @@ class ComponentLoader extends EventEmitter
 
   isGraph: (cPath) ->
     # Live graph instance
-    return true if typeof cPath is 'object' and cPath instanceof nofloGraph.Graph
+    return true if typeof cPath is 'object' and cPath instanceof fbpGraph.Graph
```

Comment:
it should work given that `noflo.Graph` is now just an alias of `fbp-graph.Graph`